packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -148,6 +148,9 @@ export class TabBarRenderer extends TabBar.Renderer {
         const style = this.createTabStyle(data);
         const className = this.createTabClass(data);
         const dataset = this.createTabDataset(data);
+        const closeIconTitle = data.title.className.includes(PINNED_CLASS)
+            ? nls.localizeByDefault('Unpin')
```

Comment:
In VS Code I noticed that his is `Unpin Editor`, but given we can pin more than just editors I'm fine with the changes :+1:

Reply:
I agree, I had this discussion with Colin over in the original PR: https://github.com/eclipse-theia/theia/pull/10817#discussion_r816709293, We decided in favor of just using `Unpin`, since we can theoretically pin anything.